### PR TITLE
Send Emails to contact@ on Important Suggestions

### PR DESCRIPTION
**Pull request by phil-lopreiato**
For now, just offseason events and api key requests

### CODE REVIEW

File: controllers/suggestions/suggest_apiwrite_controller.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 from consts.auth_type import AuthType
 from controllers.base_controller import LoggedInHandler
+from controllers.suggestions.suggestions_base_controller import SuggestionsBaseController
 from helpers.suggestions.suggestion_creator import SuggestionCreator
 from template_engine import jinja2_engine
 
 
-class SuggestApiWriteController(LoggedInHandler):
+class SuggestApiWriteController(SuggestionsBaseController):
```
Comment (fangeugene) [inline controllers/suggestions/suggest_apiwrite_controller.py]: If I understand correctly, SuggestionsBaseController is only useful to wrap the email sender. Would it make more sense to keep this LoggedInHandler and wrap the email sending in a helper?

File: controllers/suggestions/suggestions_base_controller.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def __init__(self, *args, **kw):
+        super(SuggestionsReviewBaseController, self).__init__(*args, **kw)
+        self.verify_permissions()
+
+    def verify_permissions(self):
+        for permission in self.REQUIRED_PERMISSIONS:
+            self._require_permission(permission)
+
+
+class SuggestionsBaseController(LoggedInHandler):
```
Comment (fangeugene) [inline controllers/suggestions/suggestions_base_controller.py]: I think this is kind of pointless. A helper makes more sense.

Comment (phil-lopreiato) [inline controllers/suggestions/suggestions_base_controller.py]: :+1:

**Comment by phil-lopreiato** [comment]
Moved the email code to a helper

**Comment by gregmarra** [comment]
This is awesome.

**Comment by gregmarra** [comment]
Should we email when they are acted on too? That way several people won't
go to look.

**Comment by phil-lopreiato** [comment]
Yeah, I think that's a good idea. We'll just have to make sure they stay in the same thread

**Pull request merged**